test/server/cards/03_TWI/units/HevyStaunchMartyr.spec.ts

```diff
@@ -0,0 +1,32 @@
+describe('Hevy Staunch Martyr', function () {
+    integration(function (contextRef) {
+        describe('Hevy Staunch Martyr\'s ability', function () {
+            it('should attack for 6 with raid 2 on coordinate', function () {
+                contextRef.setupTest({
+                    phase: 'action',
+                    player1: {
+                        groundArena: ['specforce-soldier', 'luke-skywalker#jedi-knight', 'hevy#staunch-martyr'],
+                    },
+                    player2: {
+                        groundArena: ['consular-security-force', 'wampa', 'fleet-lieutenant'],
+                    }
+                });
+
+                const { context } = contextRef;
+
+                context.player1.clickCard(context.hevy);
```

Comment:
can you check power before and after coordinate is active ?

Reply:
I did something. Not 100% it's what you had in mind. Let me know.